packages/back-end/src/models/UserModel.ts

```diff
@@ -15,6 +15,8 @@ const userSchema = new mongoose.Schema({
   superAdmin: Boolean,
   verified: Boolean,
   minTokenDate: Date,
+  externalId: String,
+  managedByIdp: Boolean,
```

Comment:
This is not safe to store in the UserModel since users can belong to multiple organizations.  It needs to be stored as part of the `members` array in the organization instead.